http-client/Network/HTTP/Client/Manager.hs

```diff
@@ -286,6 +287,16 @@ noProxy = ProxyOverride $ const $ return $ \req -> req { proxy = Nothing }
 useProxy :: Proxy -> ProxyOverride
 useProxy p = ProxyOverride $ const $ return $ \req -> req { proxy = Just p }
 
+-- | Send secure requests to the proxy in plain text rather than using CONNECT,
+-- regardless of the value in the @Request@.
+--
+-- Since 0.7.2
+useProxySendRequestMode :: ProxyOverride -> ProxyOverride
```

Comment:
This type signature feels a bit weird, I would have thought it would take a `Proxy` instead of `ProxyOverride`, just like `useProxy` does. Is there a specific reason to take a `ProxyOverride`.

Reply:
The intention was that it modifies `useProxy` rather than replaces it (i.e. usage would be `useProxySendRequestMode (useProxy $ Proxy ...))`). But happy to make it so you use it instead of `useProxy` and supply the Proxy directly to it.

Reply:
does this mean that there's no way to rely on the proxy set in the environment, but to also let it know that it should send requests in the clear?